OneSignalSDK/onesignal/notifications/build.gradle

```diff
@@ -69,23 +69,13 @@ dependencies {
 
     compileOnly('com.amazon.device:amazon-appstore-sdk:[3.0.1, 3.0.99]')
 
-    // play-services-base:16.1.0 is the last version before going to AndroidX
-    // play-services-base:17.0.0 is the first version using AndroidX
-    // Required for GoogleApiAvailability
-    implementation('com.google.android.gms:play-services-base') {
```

Comment:
The comment stated that play-services-base is required for GoogleApiAvailability. Will it affect the availability now that we remove the dependency? Or we leave it to the customer to add the dependency?

Reply:
Good question, it is part `firebase-messaging:23.0.8` dependencies. See it's dependencies here:
https://mvnrepository.com/artifact/com.google.firebase/firebase-messaging/23.0.8

We are safe still if Google happens to change the dependencies in the future, as we will know right away as we will get a compile error on `GoogleApiAvailability` when we build the SDK.

Reply:
I just pulled and ran some normal operations on the sample app, everything works so far. This looks good to me now!